shell/platform/darwin/ios/framework/Headers/FlutterPlatformViews.h

```diff
@@ -28,9 +38,9 @@ FLUTTER_EXPORT
  *   code, this will be null. Otherwise this will be the value sent from the Dart code as decoded by
  *   `createArgsCodec`.
  */
-- (UIView*)createWithFrame:(CGRect)frame
-            viewIdentifier:(int64_t)viewId
-                 arguments:(id _Nullable)args;
+- (NSObject<FlutterPlatformView>*)createWithFrame:(CGRect)frame
```

Comment:
Personally, I prefer `-(id<FlutterPlatformView>)`, but looks like our codebase is pretty inconsistent on this. @chinmaygarde thoughts?

Reply:
No opinion on my side 😄 
I do prefer to leave it out of this PR and have a separate one to change it in the other places in the file (or the entire codebase?) as well.